src/main.rs

```diff
@@ -351,17 +364,14 @@ fn truncated_description(description: &str) -> String {
         .split(".")
         .filter(|sentence| !sentence.trim().is_empty());
 
-    let mut out = format!(
-        "{}.",
-        lossy_unicode(&sentences.next().unwrap_or_default().as_bytes())
-    );
+    let out = sentences.next().unwrap_or_default().trim_end_matches('.');
+    let mut out = lossy_unicode(out.as_bytes());
     let mut out_len = char_len(&out);
 
     if out_len > MAX_DESCRIPTION_WIDTH {
         out = char_truncate_string(&out, MAX_DESCRIPTION_WIDTH, TRUNCATION_SUFFIX).into_owned();
     } else {
         for line in sentences {
-            let line = lossy_unicode(&line.as_bytes());
```

Comment:
Is it accurate to remove this function call? It exists here in the original Python source.

Reply:
If I am correct, we already run `lossy_unicode` when we first loads the file so I think it is unnecessary to do it again. I think it was also called above but I think that could also be removed.

Reply:
Oh, you are correct. We need to run it since it was not runned for the other parts of the description.

Reply:
I think we could also remove the `lossy_unicode` since it is always allocating a `String`, we don't need that when we do only `format!` for those usage, I use `String::from_utf8_lossy(s.as_bytes())` instead.